build.gradle

```diff
@@ -18,6 +18,7 @@ configurations {
 
 dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
```

Comment:
Was that line added by Initializr, IDE or manually?
It manually then is there any Spring site with these addresses or you've found them on Stack Overflow or something like that?

Reply:
All of the imports which are required can be found on official Spring documentation, which is the first an the last place where we should look for answers to our questions. There is documentation with example for Gradle, but also for Maven projects